contracts/core/Allo.sol

```diff
@@ -263,6 +279,20 @@ contract Allo is
         _revokeRole(pools[_poolId].managerRole, _manager);
     }
 
+    /// @notice Remove multiple pool managers
+    /// @dev Emits 'RoleRevoked()' event. 'msg.sender' must be a pool admin.
+    /// @param _poolId ID of the pool
+    /// @param _managers The addresses to remove
+    function removePoolManagers(uint256 _poolId, address[] memory _managers) external onlyPoolAdmin(_poolId) {
```

Comment:
Same thing with `addPoolManagers`

Reply:
I did it only with `addPoolManagers` since `removePoolManagers` is only one line and it will not make difference.